components/tools/OmeroWeb/build.xml

```diff
@@ -59,6 +59,21 @@
                 </touch>
             </then>
         </if>
+        <if><not><available file="${basedir}/../target/lib/python/clogging"/></not>

```

Comment:
It's a problem, and both `lib/python` and `lib/fallback` are on `PYTHONPATH`. It's just that `fallback` is added to the end, so if someone wanted to update `ConcurrentLogHandler` locally, they could.